docs/pages/server-access/guides/openssh.mdx

```diff
@@ -117,11 +146,44 @@ host.
 
 ### Issue a host certificate
 
-On your local machine, assign the IP address or fully qualified domain name of
-your Node to an environment variable.
+When creating host certificates, it is important to specify all the domain names
+and addresses that refer to your Node. If you try to connect to a Node with a 
+name or address that was not specified when creating it's host certificate,
+Teleport will reject the SSH connection.
+
+#### Finding your Node's UUID
+
+When you created a node resource earlier, a unique UUID was generated for that
```

Comment:
```suggestion
When you created a `node` resource earlier, the Teleport Auth Service generated a universal unique identifier (UUID) for that
```
- Using code style when referring to a configuration resource
- Using active voice for clarity
- Removing the repetition in "unique UUID"

Reply:
Done